xExtension-QuickCollapse/i18n/en/gen.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return [
+	'js' => [
+		'toggle_collapse' => 'Toggle collapse',
```

Comment:
Seems a bit unclear, although I suppose you'd be well aware what the extension does from the description. ^_^